docs/advanced-features/package-events.md

```diff
@@ -0,0 +1,105 @@
+# Package events
+
+Every time package revision is created in one of the Quilt stack buckets, Quilt stack
+emits event to default
```

Comment:
@akarve 
We were thinking that we need to create a bus in the stack for this, maybe we should do this before publishing docs?